editor.planx.uk/src/@planx/components/Pay/Editor.test.tsx

```diff
@@ -107,10 +106,10 @@ describe("Pay component - Editor Modal", () => {
       expect(getByDisplayValue("myValue")).toBeInTheDocument();
     });
 
-    it("allows new values to be added", async () => {
+    it("allows new values to be added", { timeout: 20000 }, async () => {
```

Comment:
rather than a single `jest.setTimeout(2000)` at the top of the file, I've gone for per-test timeouts as it seems useful/helpful to keep a more specific pulse on which frontend interactions are slow running! 

if preferred, we still have a per-file option of `vi.setConfig({ testTimeout: 20000 })`

Reply:
This approach works well for me! Easier to come back and revisit a test at a time, than a whole suite at a time.